### ISSUE TITLE
loss（pre-train）

### ISSUE THREAD

**Issue by QUANmoxiansheng** [liang3588/MeshMAE#11]
Thank you for sharing such excellent work. I have a few questions regarding pre-training. We encountered the following situation during training—does it affect the overall training?
**Epoch 2/99 [Pretrain]: 100%|███████████████████| 3073/3073 [2:21:16<00:00,  2.76s/it, loss=39307117.633155, lr=0.000100]
Epoch 2 Pretrain | Loss: 39307117.633155 | Best Loss: 2.271230 | Best Epoch: 0 | Time: 2.4h**
